packages/arb-token-bridge-ui/src/components/TransferPanel/TokenDepositCheckDialog.tsx

```diff
@@ -110,8 +57,35 @@ export function TokenDepositCheckDialog(props: TokenDepositCheckDialogProps) {
   }, [type, symbol])
 
   return (
-    <Dialog {...props} title={title}>
-      <p className="pt-4">{textContent}</p>
+    <Dialog {...props} title={title} className="flex flex-col gap-4">
+      <p>{textContent}</p>
+      <NoteBox variant="error">
+        <p>
+          <b>Do not bridge</b> if your token does something non-standard like
+          generates passive interest or is a rebasing stablecoin.
+        </p>
+        <p>
+          Not sure if your token is compatible? Check the{' '}
```

Comment:
Figma says there's a break between these two sentences but imo it looks much better inline.